### ISSUE TITLE
Router scheme required parameters

### ISSUE THREAD

**Issue by andrew-snytko** [fastify/fastify#554]
Hi!
How can I set parameter as "required" in route scheme ? It can be "querystring" or "body" or "params" parameter.

**Comment by mcollina** [comment]
I don't understand your question, can you rephrase, making an example?

**Comment by andrew-snytko** [comment]
Ok.
For example route scheme.
{
        schema: {
            querystring: {
                id: {
                    type: 'integer',
                    required: ['id']
                }
            }
    }
Does in this scheme "id" will validate as required ?
Or i have to use another syntax to validate required parameters ?

**Comment by delvedor** [comment]
Hi @andrew-snytko, I suggest you to read [this](https://github.com/fastify/fastify/blob/master/docs/Validation-and-Serialization.md#resources) resources! :)

Below what you are trying to accomplish:
```js
{
  schema: {
    querystring: {
      type: 'object',
      properties: {
         id: { type: 'integer' }
      },
      required: ['id']
    }
  }
}
```

**Comment by andrew-snytko** [comment]
Thank you for the explanation :)
